themes/default/content/blog/aws-enterprise-container-management/index.md

```diff
@@ -0,0 +1,76 @@
+---
+title: AWS Enterprise Container Management with Pulumi
+
+# The date represents the post's publish date, and by default corresponds with
+# the date this file was generated. Posts with future dates are visible in development,
+# but excluded from production builds. Use the time and timezone-offset portions of
+# of this value to schedule posts for publishing later.
+date: 2022-04-14
+
+# Use the meta_desc property to provide a brief summary (one or two sentences)
+# of the content of the post, which is useful for targeting search results or social-media
+# previews. This field is required or the build will fail the linter test.
+meta_desc: We're excited to be launch partners for the new Enterprise Container Management category of the AWS Container Competency program.
+
+# The meta_image appears in social-media previews and on the blog home page.
+# A placeholder image representing the recommended format, dimensions and aspect
+# ratio has been provided for you.
+meta_image: meta.png
+
+# At least one author is required. The values in this list correspond with the `id`
+# properties of the team member files at /data/team/team. Create a file for yourself
+# if you don't already have one.
+authors:
+    - isaac-harris
+
+# At least one tag is required. Lowercase, hyphen-delimited is recommended.
+tags:
+    - aws
+    - containers
+    - kubernetes
+
+# See the blogging docs at https://github.com/pulumi/pulumi-hugo/blob/master/BLOGGING.md.
+# for additional details, and please remove these comments before submitting for review.
+---
+
+Managing containers and Kubernetes clusters are consistently popular topic areas on the Pulumi blog and in our docs. Our customers regularly cite that Pulumi simplifies container management scenarios, making it the primary reason for choosing Pulumi to define, deploy and manage all of their cloud resources. This includes teams that are just starting their cloud journey and spinning up their first project, as well as teams that want to modernize their apps and services with cloud-native architectures or even scale from one to many clouds.
+
+We’re excited to be launch partners for the new [Enterprise Container Management Solutions category](https://aws.amazon.com/blogs/apn/aws-container-competency-expands-to-include-enterprise-container-management-category) of the AWS Container Competency program because it perfectly encapsulates Pulumi’s capabilities across the entire lifecycle of container-based architectures and applications. These scenarios are critical to the success of every cloud engineering and DevOps team and include provisioning, governance, security, and observability across [Amazon Elastic Kubernetes Service](https://aws.amazon.com/eks) (EKS) and [Amazon Elastic Container Service](https://aws.amazon.com/ecs) (ECS). Here is a handy guide to some of the features that make Pulumi a great Enterprise Container Management Solution and how Pulumi makes each container scenario easier for your team.
```

Comment:
```suggestion
We’re excited to be launch partners for the new [Enterprise Container Management category](https://aws.amazon.com/blogs/apn/aws-container-competency-expands-to-include-enterprise-container-management-category) of the AWS Container Competency program because it perfectly encapsulates Pulumi’s capabilities across the entire lifecycle of container-based architectures and applications. These scenarios are critical to the success of every cloud engineering and DevOps team and include provisioning, governance, security, and observability across [Amazon Elastic Kubernetes Service](https://aws.amazon.com/eks) (EKS) and [Amazon Elastic Container Service](https://aws.amazon.com/ecs) (ECS). Here is a handy guide to some of the features that make Pulumi a great Enterprise Container Management Solution and how Pulumi makes each container scenario easier for your team.
```